config_complete.toml

```diff
@@ -315,3 +315,44 @@ MetricsReportingInterval = 3
 # listener.
 # Not eligible for live reload.
 # MetricsListenAddr = "localhost:2112"
+
+###########################
+## gRPC ServerParameters ##
+###########################
+
+# Reflects: https://pkg.go.dev/google.golang.org/grpc/keepalive#ServerParameters
+
+[GRPCServerParameters]
+
+# MaxConnectionIdle is a duration for the amount of time after which an
+# idle connection would be closed by sending a GoAway. Idleness duration is
+# defined since the most recent time the number of outstanding RPCs became
+# zero or the connection establishment.
+# Not eligible for live reload.
+# MaxConnectionIdle = "1m"
+
+# MaxConnectionAge is a duration for the maximum amount of time a
+# connection may exist before it will be closed by sending a GoAway. A
+# random jitter of +/-10% will be added to MaxConnectionAge to spread out
+# connection storms.
+# 0s sets duration to infinity which is the default.
+# Not eligible for live reload.
+# MaxConnectionAge = "0s"
+
+# MaxConnectionAgeGrace is an additive period after MaxConnectionAge after
+# which the connection will be forcibly closed.
+# 0s sets duration to infinity which is the default.
+# Not eligible for live reload.
+# MaxConnectionAgeGrace = "0s"
+
+# After a duration of this time if the server doesn't see any activity it
+# pings the client to see if the transport is still alive.
+# If set below 1s, a minimum value of 1s will be used instead.
```

Comment:
According to the [docs here](https://pkg.go.dev/google.golang.org/grpc@v1.49.0/keepalive#ClientParameters), the lower limit is 10s, not 1s, unless I'm reading the wrong thing. Can you check please, and also note the meaning of 0s?

Reply:
Thanks for pointing that out @kentquirk! Although, it looks like what you linked to is for the `ClientParameters`.

I grabbed the comments here for the [ServerParameters](https://github.com/grpc/grpc-go/blob/v1.49.0/keepalive/keepalive.go#L66).

I've also pushed a commit for adding notes about the meaning of 0s!